src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java

```diff
@@ -209,4 +209,4 @@ public void testGetLastRevisionsQueryAfterDeletion() throws IOException, Interru
     Assert.assertEquals(results.get(0), dummy2);
     Assert.assertEquals(results.get(1), dummy1);
   }
-}
+}
```

Comment:
[add new line to the end of file](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,you%20have%20reached%20the%20end.)

Reply:
File reverted to master. line space should be preserved.